config/jobs/kubernetes/sig-cloud-provider/azure/release-master.yaml

```diff
@@ -96,122 +98,116 @@ presubmits:
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
     extra_refs:
-    - org: kubernetes-sigs
-      repo: azuredisk-csi-driver
-      base_ref: master
-      path_alias: sigs.k8s.io/azuredisk-csi-driver
+      - org: kubernetes-sigs
+        repo: cluster-api-provider-azure
+        base_ref: master
+        path_alias: sigs.k8s.io/cluster-api-provider-azure
+        workdir: true
+      - org: kubernetes-sigs
+        repo: azurefile-csi-driver
+        base_ref: master
+        path_alias: sigs.k8s.io/azurefile-csi-driver
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210601-ea6aa4e-master
-        command:
-        - runner.sh
-        - kubetest
-        args:
-        # Generic e2e test args
-        - --test
-        - --up
-        - --down
-        - --build=quick
-        - --dump=$(ARTIFACTS)
-        # Azure-specific test args
-        - --provider=skeleton
-        - --deployment=aksengine
-        - --aksengine-admin-username=azureuser
-        - --aksengine-creds=$(AZURE_CREDENTIALS)
-        - --aksengine-orchestratorRelease=1.22
-        - --aksengine-deploy-custom-k8s
-        - --aksengine-public-key=$(AZURE_SSH_PUBLIC_KEY_FILE)
-        - --aksengine-template-url=https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/master/test/e2e/manifest/in-tree-vmss.json
-        - --aksengine-download-url=https://github.com/Azure/aks-engine/releases/download/nightly/aks-engine-nightly-linux-amd64.tar.gz
-        # Specific test args
-        - --test-azure-disk-csi-driver
-        securityContext:
-          privileged: true
-        env:
-        - name: AZURE_STORAGE_DRIVER
-          value: "kubernetes.io/azure-disk" # In-tree Azure disk storage class
-        - name: ENABLE_TOPOLOGY
-          value: "true"
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20210601-ea6aa4e-master
+          command:
+            - runner.sh
+            - ./scripts/ci-entrypoint.sh
+          args:
+            - bash
+            - -c
+            - >-
+              kubectl apply -f templates/addons/azurefile-role.yaml &&
+              cd ${GOPATH}/src/sigs.k8s.io/azurefile-csi-driver &&
+              make e2e-test
+          env:
+            - name: AZURE_STORAGE_DRIVER
+              value: kubernetes.io/azure-file # In-tree Azure file storage class
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              cpu: 1
+              memory: "4Gi"
     annotations:
       testgrid-dashboards: provider-azure-presubmit
-      testgrid-tab-name: pull-kubernetes-e2e-aks-engine-azure-disk-vmss
+      testgrid-tab-name: pull-kubernetes-e2e-capz-azure-file
       testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
       testgrid-num-columns-recent: '30'
-  - name: pull-kubernetes-e2e-aks-engine-azure-file
+  - name: pull-kubernetes-e2e-capz-azure-file-machinepool
```

Comment:
should we name this vmss instead since that's what is tested (via machinepools) and might be more recognizable by folks with no capz context?

Reply:
renamed to vmss